kie-server-parent/kie-server-router/kie-server-router-proxy/src/main/java/org/kie/server/router/identity/FileStoredIdentityManager.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.server.router.identity;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+import io.undertow.security.idm.Account;
+import io.undertow.security.idm.Credential;
+import io.undertow.security.idm.PasswordCredential;
+import org.jboss.logging.Logger;
+
+public class FileStoredIdentityManager implements IdentityService {
+
+    protected static final Logger log = Logger.getLogger(FileStoredIdentityManager.class);
+
+    private File userFile;
+
+    public FileStoredIdentityManager() {
+        userFile = new File("users.properties");
```

Comment:
Would it be possible to make location of users.properties configurable?

Reply:
Something similar to https://github.com/kiegroup/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-router/kie-server-router-proxy/src/main/java/org/kie/server/router/repository/FileRepository.java#L39